rfd/00XX-agent-census.md

```diff
@@ -0,0 +1,234 @@
+---
+authors: Vitor Enes (vitor@goteleport.com)
+state: draft
+---
+
+# RFD XX - Agent Census
+
+## Required Approvals
+
+* Engineering: @rjones && @jimbishopp
+* Product: @xin || @klizhentas
+
+## What
+
+This RFD details how we'll track more information about agents (aka Agent Census).
+A brief description of this task can be found in [Cloud's RFD 53](https://github.com/gravitational/cloud/tree/54559795b45b2e8515ea7e159d26cadfbb52482f/rfd/0053-prehog.md).
+
+#### Goals
+
+- Track more information about each Teleport agent.
```

Comment:
Nit: vague, but spelled out really well in Details > Data Tracked

Reply:
Tried to improve this in 3b0e0c38c5eb0bdeb87fa79506aad4990d004e52.